### ISSUE TITLE
`Pagination` violates `landmark-unique` accessibility rule when used more than once within a page

### ISSUE THREAD

**Issue by mcmxcdev** [melt-ui/melt-ui#1160]
### Describe the bug

https://melt-ui.com/docs/builders/pagination example uses `nav` HTML tag as part of the code snippet.

When using a Chrome extension like [axe](https://chromewebstore.google.com/detail/axe-devtools-web-accessib/lhdoppojpmngadmnindnejefpokejbdd) to go over a page including more than one usage of `Pagination`, there will be a violation of https://dequeuniversity.com/rules/axe/4.8/landmark-unique?application=AxeChrome 

`nav` HTML tag is only allowed once per page OR needs to conform to "The landmark must have a unique aria-label, aria-labelledby, or title to make landmarks distinguishable"

I understand that this can be solved in userland, but if its possible to solve it in melt-ui then that doesn't even become a potential issue for every project consuming it. 

### Reproduction

-

### System Info

```bash
-
```

### Severity

annoyance

**Comment by huntabyte** [comment]
This is something other than what Melt should handle. You should provide an aria label to the `nav` element that adds context to the navigation. Otherwise, we'd generate a random label that would pass the axe tests but not provide anything meaningful to a user of those labels.
